src/styles/header.scss

```
@@ -0,0 +1,132 @@
+@import "utils";
+
+.head {
+  background-image: url(../images/header.jpg);
+  background-size: cover;
+  background-blend-mode: multiply;
+  background-color: rgba(255, 0, 0, 0.2);
+}
+
+.header {
+  height: 100vh;
+  width: 100%;
+  margin-bottom: 100px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  @extend %main-text;
+
+  &__title {
+    margin-top: 40vh;
+  }
+
+  &__title-text {
+    font-size: 60px;
+    line-height: 60px;
+    letter-spacing: -1.5px;
+    text-transform: none;
+    font-weight: bold;
+    position: relative;
+
+    @media (max-width: 800px) {
+      font-size: 48px;
+      line-height: 40px;
+    }
+  }
+
+  &__title-text::after {
+    content: "";
+    position: absolute;
+    top: -15px;
+    left: 2px;
+    width: 35px;
+    height: 2px;
+    background-color: white;
+    border-radius: 2px;
+  }
+
+  @media (max-width: 800px) {
+    .header {
+      padding: 24px 20px;
+    }
+  }

```

в css будет такой селекор  `.header .header`
```suggestion
  @media (max-width: 800px) {
      padding: 24px 20px;
  }

```